mapper.py

```diff
@@ -0,0 +1,275 @@
+import csv
+import os
+import shutil
+from collections import OrderedDict
+
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+TMP_DIR = os.path.join(BASE_DIR, 'tmp')
+NEW_DIR = os.path.join(TMP_DIR, 'archived')
+
+def get_new_columns():
```

Comment:
This mapper can really benefit by generalizing everything.

The `get_new_columns` functionality should read from a configuration file that is a mapping. This could be a `json` file.